docs/html/_sources/install.rst.txt

```diff
@@ -0,0 +1,105 @@
+sncli
+#####
+Simplenote Command Line Interface
+
+.. image:: screenshot1.png
+    :width: 90%
+    :align: center
+
+
+Console GUI Features
+====================
+
++ full two-way sync with Simplenote performed dynamically in the background
++ all actions logged and easily reviewed
++ list note titles (configurable format w/ title, date, flags, tags, keys, etc)
++ sort notes by date, alpha by title, tags, pinned on top
++ search for notes using a Google style search pattern or Regular Expression
++ view note contents and meta data
++ view and restore previous versions of notes
++ pipe note contents to external command
++ create and edit notes (using your editor)
++ edit note tags
++ trash/untrash notes
++ pin/unpin notes
++ flag notes as markdown or not
++ vi-like keybinds (fully configurable)
++ Colors! (fully configurable)
+
+Command Line Scripting
+======================
+
++ force a full two-way sync with Simplenote
++ all actions logged and easily reviewed
++ list note titles and keys
++ search for notes using a Google style search pattern or Regular Expression
++ dump note contents
++ create a new note (via stdin or editor)
++ import a note with raw json data (stdin or editor)
++ edit a note (via editor)
++ trash/untrash a note
++ pin/unpin a note
++ flag note as markdown or not
++ view and edit note tags
+
+TLDR
+====
+
+Notes can be viewed/created/edited in both an online and offline mode. All changes are saved to a local cache on disk and automatically sync'ed when sncli is brought online.
+
+.. note:: GitHub `pull requests`_ are welcome!
+
+.. _pull requests: https://github.com/insanum/sncli/pulls
+
+-----
+
+Install
+#######
+
+Prerequisites
+=============
+
++ A love for the command line
++ `Python 3`_
++ `Pip`_
+
+.. sidebar:: The Python modules `Urwid`_, `Requests`_, and `Simperium3`_ will also be installed.
+
+Installation
+=============
+
+Pip
+---
+.. code-block:: shell
+
+    $ pip install sncli
+
+Arch AUR
+----------
+.. code-block:: shell
+
+    $ yaourt -S sncli-git
```

Comment:
yaourt is dead and we should not suggest this. Instead probable better to just say that it's available on the aur as `sncli-git`

Reply:
> One final point - can we use a built-in sphinx theme? I know this theme is nice, but then we need to maintain it and there are a lot of extra files in the repo now... :/

As far as I can see, any theme used, including the default `alabaster` requires you copy some files into the `_static` folder in order for the theme to be applied. I've only been playing with Sphinx for four days, there is a good chance I am missing something...

Reply:
> yaourt is dead and we should not suggest this. Instead probable better to just say that it's available on the aur as `sncli-git`

Yes--I didn't realize `yaourt` is dead (I still use it)--this can be changed.